package.json

```diff
@@ -24,13 +25,38 @@
   },
   "homepage": "https://github.com/Mike-Stupich/typed3#readme",
   "dependencies": {
+    "@types/jest": "22.2.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
```

Comment:
Should be in dev dependencies

Reply:
Not sure if this is actually needed though

Reply:
TravisCI build was breaking until I added it.